src/handlers/commands.py

```
@@ -133,9 +98,9 @@ async def cb_back(c: CallbackQuery, state: FSMContext):
         await state.clear()
         await delete_all_bot_messages(c.from_user.id, bot_instance)
         try:
-            msg = await c.message.edit_text(T["start"], reply_markup=kb_main())
+            msg = await c.message.edit_text(T["start"].format(version=BOT_VERSION), reply_markup=kb_main())
         except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)